src/index.html

```diff
@@ -7,7 +8,57 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Stars</h1>
+    <section class="section">
```

Comment:
The section in this task does nothing. Let's have a rule for the future, that we never add anything to the code, that does nothing. Remove this tag and styles for the class section